actioncable/lib/action_cable/connection/web_socket_rack.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+module ActionCable
+  module Connection
+    # Wrap the real socket to minimize the externally-presented API
+    #
+    # This variation leverages the `rack.upgrade?` approach detailed here: https://github.com/rack/rack/pull/1272
+    class WebSocketRack # :nodoc:
+      UPGRADE_EXISTS = "rack.upgrade?".freeze
```

Comment:
Shouldn't need `.freeze` at the end of strings with `# frozen_string_literal: true` at the top of your file.

Reply:
True... I guess old habits catch up with me.

Thanks! 👍🏻